Python/pywarpx/picmi.py

```diff
@@ -100,7 +100,7 @@ def initialize_inputs(self, layout,
         self.species = pywarpx.Bucket.Bucket(self.name,
                                              mass = self.mass,
                                              charge = self.charge,
-                                             injection_style = 'python',
+                                             injection_style = 'none',
```

Comment:
This could also be set to `None`

Reply:
@dpgrote Thanks, I just did this. Should be good now.